db/version_set.cc

```diff
@@ -4164,9 +4154,13 @@ void VersionSet::Reset() {
   if (column_family_set_) {
     WriteBufferManager* wbm = column_family_set_->write_buffer_manager();
     WriteController* wc = column_family_set_->write_controller();
+    // db_id becomes the source of truth after DBImpl::Recover():
+    // https://github.com/facebook/rocksdb/blob/main/db/db_impl/db_impl_open.cc#L527.
```

Comment:
This link may be invalid because updates are added to the main branch. Maybe use an existing release branch.

Reply:
Done, thanks @riversand963 !!